docs/source/start.rst

```diff
@@ -189,20 +190,23 @@ variables ``FOREST_CONFIG`` and ``QCS_CONFIG``, or else at the respective defaul
     [QPU]
     exec_on_engage = :
 
```

Comment:
Can we triple check that the content of these files is correct? I only mention this because my `~/.forest_config` and `~/.qcs_config` on my QMI look different from these.